.github/workflows/lint-and-test.yml

```
@@ -54,6 +54,28 @@ jobs:
             - name: Lint PHP
               run: composer lint
 
+    phpstan:
+        name: PHPStan
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 'latest'
+                  coverage: none
+                  tools: composer, cs2pr
+
+            - name: Install PHP dependencies
+              uses: ramsey/composer-install@v2
+              with:
+                  composer-options: '--prefer-dist --no-progress --no-interaction'
+
+            - name: Run PHPStan
+              run: composer phpstan
```

This is a script, not a command.
```suggestion
              run: composer run phpstan
```

---

It works though, but makes sense to be explicit 👍